plugin/trino-hive/src/main/java/io/trino/plugin/hive/rcfile/RcFilePageSourceFactory.java

```diff
@@ -193,9 +193,4 @@ else if (deserializerClassName.equals(COLUMNAR_SERDE_CLASS)) {
             throw new TrinoException(HIVE_CANNOT_OPEN_SPLIT, message, e);
         }
     }
-
-    private static String splitError(Throwable t, Location path, long start, long length)
```

Comment:
This same message is used in several other places. Please update all of them to use the new method.